open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventProducer.java

```diff
@@ -206,11 +244,19 @@ public void run()
             catch (Throwable   error)
             {
                 log.error("Bad exception from sending events " + error.getMessage());
-                this.recoverAfterError();
+
+                if( isExceptionRetryable(error) ) {
+                    this.recoverAfterError();
```

Comment:
I noticed in this method we are using time in 2 different units - around line 39. Seconds for recoverytime, milliseconds for regular sleep time. though handled correctly would be better to use consistent units?

Reply:
@planetf1 again this code was there , but I'm happy to change it if Mandy agrees.

Reply:
Good to stick with this for our immediate kafka fix - can consider revisiting with a new connector implementation